workers/blobuploadcleanupworker/blobuploadcleanupworker.py

```diff
@@ -28,13 +28,31 @@ def __init__(self):
     def _try_cleanup_uploads(self):
         """
         Performs garbage collection on the blobupload table.
+        Will also perform garbage collection on the uploads folder in the S3 bucket,
+        if applicable.
         """
         try:
             with GlobalLock("BLOB_CLEANUP", lock_ttl=LOCK_TTL):
                 self._cleanup_uploads()
         except LockNotAcquiredException:
             logger.debug("Could not acquire global lock for blob upload cleanup worker")
-            return
+
+        if "FEATURE_CLEAN_UPLOADS_FOLDER" in app.config:
```

Comment:
You need to add this config flag in `config.py` (set it to default to `False`).
Also, any config flag prefixed with `FEATURE_` gets added to the global `features` object. So in this case it would look like:
```
if features.CLEAN_UPLOADS_FOLDER:
  ...
```

Reply:
This should not be prefixed with `FEATURE_`, since it's a "subfeature" of the blobuploadcleanupworker.
Probably something just  `CLEAN_BLOB_UPLOAD_FOLDER` is enough, and then can be referred like:
```
if app.config.get("CLEAN_BLOB_UPLOAD_FOLDER", False):
   ...
```

Reply:
Understood. I updated the flag to be `CLEAN_BLOB_UPLOAD_FOLDER` and added `CLEAN_BLOB_UPLOAD_FOLDER = False` to config.py. I also updated the condition to use `if app.config.get("CLEAN_BLOB_UPLOAD_FOLDER", False):`